.github/workflows/actions.yml

```diff
@@ -14,6 +13,8 @@ env:
   DOTNET_NOLOGO: true
 
 jobs:
+  variables:
+    REMOVE_BUILD_TOOL_VERSION: "31.0.0"
```

Comment:
could that variable be put below in the android sample section?

Reply:
Sure! Would you happen to know how to define a variable in the android sample section? I'm not sure how to do it correctly :)

Reply:
No idea :) That was more a question than an affirmation